i18n/enu/out/debug.i18n.json

```diff
@@ -1,5 +1,5 @@
 {
-  "autolispext.debug.nodap": "doesn’t exist. Verify that the file exists in the same folder as that for the product specified in the launch.json file.",
+  "autolispext.debug.nodap": "doesn’t exist. Use a release later than AutoCAD 2020. Verify that the file exists in the same folder with the product to debug.",
```

Comment:
Vscode debug supports AutoCAD 2021 and onward

Reply:
This is a copy of existing statement here:
https://github.com/Autodesk-AutoCAD/AutoLispExt/pull/137/files#diff-cb5c8ca542958828c6432e00b457edb26ac115c3f63e24d6cd54d98c548732dcR4

Reply:
Can we also update the readme.md as well?

Reply:
@nigma456 it's already the 2nd sentence in readme:
![image](https://user-images.githubusercontent.com/57521938/114958429-90b48f00-9e95-11eb-9503-7e2b3175f587.png)

Reply:
Yes, I know. I mean AutoCAD 2021 and onward